app/models/server/raw/Rooms.js

```diff
@@ -3,6 +3,15 @@ import { escapeRegExp } from '@rocket.chat/string-helpers';
 import { BaseRaw } from './BaseRaw';
 
 export class RoomsRaw extends BaseRaw {
+	indexes=[
```

Comment:
@cauefcr This index was not applied when I restarted my local server. I feel this has something to do with this file still being in .js instead of .ts :thinking: 
Anyways, when I applied the same index in [this](https://github.com/RocketChat/Rocket.Chat/blob/7660464d086ecdaef18cf3bd9eae7684ff317f18/app/models/server/models/LivechatRooms.js#L23) file, I could see that it was properly applied. So maybe we need to move this over there. @KevLehman WDYT? Can you check if the indexes are getting applied on your local?

Reply:
weird, it got applied on mine, i'll move it over.